### PR TITLE
[EWL-5456] SG2 | Create "Member Feature" Organism

### PR DESCRIPTION
**Pull request by csj1012**
## Ticket(s)

**Jira Ticket**
- [EWL-5456: SG2 | Create "Member Feature" Organism](https://issues.ama-assn.org/browse/EWL-5456)

## Description
- Adds the Member Feature organism.
- Adds the organism to the home page.
- Corrects a typo in the Section Title atom.

## To Test
- [ ] `gulp serve`
- [ ] Given that I am viewing SG2
- [ ] When I click "Organisms"
- [ ] Then I should see the "Section title" atom
- [ ] And I should see the "Hero story" molecule
- [ ] And I should see the "Hero Feature rail" molecule
- [ ] And the "Hero Feature Rail" should be to the right of the "Hero Story"
- [ ] When I click "Pages" in the menu
- [ ] Then I should see "Homepage"
- [ ] When I click "Homepage"
- [ ] Then I should see "Hero Feature"
- [x] Did you test in IE 11?

## Visual Regressions

A report is available [345](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/[PULL_REQUEST_ID]/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.

## Relevant Screenshots/GIFs
screen shot 2018-07-13 at 8 56 00 am
screencapture-localhost-3000-patterns-03-organisms-member-feature-03-organisms-member-feature-html-2018-07-13-08_55_43

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by designerneil** [comment]
Your testing instructions are unclear:

- [ ] When I click "Organisms"
- [ ] Then I should see the "Section title" atom
- [ ] And I should see the "Hero story" molecule
- [ ] And I should see the "Hero Feature rail" molecule


I think you meant to say:
When I click "Organisms"
Then I should see the "Member Feature"
And in the Member Feature Organism I should see "Hero story" molecule
And in the Member Feature Organism I should see  "Hero Feature rail" molecule

**Comment by designerneil** [comment]
The gap between the h1 and the author name should be less
screen shot 2018-07-13 at 10 38 19 am

**Comment by designerneil** [comment]
- [ ] When I click "Pages" in the menu
- [ ] Then I should see "Homepage"
- [ ] When I click "Homepage"
- [ ] Then I should see "Hero Feature"

I don't see the  "Hero Feature" did you mean Membership Feature?

**Comment by designerneil** [comment]
The width of the white box with the quote seems to be smaller than the image below in Zeplin mockup
screen shot 2018-07-13 at 10 45 12 am

**Comment by designerneil** [comment]
On tablet the width of the right column is too small causing the words to wrap weirdly.
The h1 and quote does not match the mockup
The view more should not be right aligned
The spacing between the items in the right column too close together

**Curent implementation:**
screen shot 2018-07-13 at 10 50 02 am

**Zeplin**
screen shot 2018-07-13 at 10 50 23 am

**Comment by designerneil** [comment]
On mobile the section title should not span the full width of the screen and needs more padding around it

**From zeplin**
screen shot 2018-07-13 at 10 53 02 am
 and should be uppercase.

**Comment by designerneil** [comment]
On mobile: ama__member-feature-rail the purple name should have more space around it.

**From zeplin**
screen shot 2018-07-13 at 10 54 38 am

File: styleguide/source/_patterns/03-organisms/member-feature-rail.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+
+### Variables
+~~~~
+{
+  "memberFeatureRail": {
+    "articleStubs": [
+      {
+        "hasVideo": 
+          string / required ("true")
```
Comment (designerneil) [inline styleguide/source/_patterns/03-organisms/member-feature-rail.md]: boolean

**Comment by designerneil** [comment]
I assume this was branched off of:
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/344

Please note if this was branched off another branch or has dependencies

**Comment by JaeBea** [comment]
@labbydev I have updated some styles and resolved merge conflicts.

**Review by labbydev — APPROVED** [review]

**Pull request merged**